NETDefault/UnitTestingWithNUnit&Moq/ServiceTestsBase.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Moq;
+using Moq.AutoMock;
+using NUnit.Framework;
+
+namespace ServiceTests
+{
+    internal class ServiceTestsBase
+    {
+        protected Service _subject;
+        protected Mock<IGateway> _gateway;
+
+        protected string knownUsername = "known";
+        protected string unknownUsername = "unknown";
+        protected int userId = 2;
+        protected string[] hobbies = new string[3] { "eat", "sleep", "repeat" };
```

Comment:
These hardcoded hobbies will make it hard to test different scenarios. Why not accept a hobbies parameter at `SetupGetHobbiesByUserId`? that will make it easier, and more visual.